File: Servers/Python/Aiohttp/routers/db_sleep.py
from aiohttp import web

from Servers.Python.utils.db.sql_query import *
from Servers.Python.utils.path import DbSleep

router = web.RouteTableDef()


@router.get(DbSleep.zero_25_second_sleep)
async def zero_25_second_sleep(request: web.Request) -> web.Response:
    async with request.app['pool'].acquire() as con:
        await con.fetch(
            QUERY_SLEEP_0_25
        )
        return web.json_response({'hello': 'world'})


@router.get(DbSleep.zero_5_second_sleep)
async def zero_5_second_sleep(request: web.Request) -> web.Response:
    async with request.app['pool'].acquire() as con:
        await con.fetch(
            QUERY_SLEEP_0_5
        )
        return web.json_response({'hello': 'world'})


@router.get(DbSleep.zero_75_second_sleep)
async def zero_75_second_sleep(request: web.Request) -> web.Response:
    async with request.app['pool'].acquire() as con:
        await con.fetch(
            QUERY_SLEEP_0_75
        )
        return web.json_response({'hello': 'world'})


@router.get(DbSleep.one_second_sleep)
async def one_second_sleep(request: web.Request) -> web.Response:
    async with request.app['pool'].acquire() as con:
        await con.fetch(
            QUERY_SLEEP_1
        )
        return web.json_response({'hello': 'world'})


@router.get(DbSleep.two_second_sleep)
async def two_second_sleep(request: web.Request) -> web.Response:
    async with request.app['pool'].acquire() as con:
        await con.fetch(
            QUERY_SLEEP_2
        )
        return web.json_response({'hello': 'world'})

File: Servers/JS/Express/routers/dbSleep.js
const router = require("express").Router();

const { pool } = require("../db");
const {
  QUERY_SLEEP_0_25,
  QUERY_SLEEP_0_5,
  QUERY_SLEEP_0_75,
  QUERY_SLEEP_1,
  QUERY_SLEEP_2,
} = require("../../db/query");

const { dbSleep } = require("../../utils/paths");

router.get(dbSleep.zero25SecondSleep, async (_, res) => {
  await pool.query(QUERY_SLEEP_0_25);
  res.send({ Hello: "World" });
});
router.get(dbSleep.zero5SecondSleep, async (_, res) => {
  await pool.query(QUERY_SLEEP_0_5);
  res.send({ Hello: "World" });
});
router.get(dbSleep.zero75SecondSleep, async (_, res) => {
  await pool.query(QUERY_SLEEP_0_75);
  res.send({ Hello: "World" });
});
router.get(dbSleep.oneSecondSleep, async (_, res) => {
  await pool.query(QUERY_SLEEP_1);
  res.send({ Hello: "World" });
});
router.get(dbSleep.twoSecondSleep, async (_, res) => {
  await pool.query(QUERY_SLEEP_2);
  res.send({ Hello: "World" });
});

module.exports = router;
